JuliaInterface/src/gap_macros.c

```diff
@@ -1,13 +1,14 @@
 #include "libgap-api.h"
 
+jl_module_t* get_module_from_string(char*);
```

Comment:
Huh, doesn't this PR *remove* get_module_from_string ?

Reply:
No, it just moves it all the way to the top.

Reply:
OK, but then why is this decl needed, considering that the function implementation now is moved to before the `#include "gap_macros.c"`?